CHANGELOG-18.0.md

```diff
@@ -36,4 +36,174 @@ You can see all the changes done to the package that you carry about with this t
+- minor fixes in elfinder in grapesjs by @grossmannmartin in https://github.com/shopsys/shopsys/pull/4232
+- fixed group admin form fields in cards by @grossmannmartin in https://github.com/shopsys/shopsys/pull/4261
+- update administartion styles by @chlebektomas in https://github.com/shopsys/shopsys/pull/4260
+- update states after form submissionm by @chlebektomas in https://github.com/shopsys/shopsys/pull/4303
+- redesign contact informations by @chlebektomas in https://github.com/shopsys/shopsys/pull/4221
+- reset password layout by @chlebektomas in https://github.com/shopsys/shopsys/pull/4254
+- comparison and wishlist tweaks by @chlebektomas in https://github.com/shopsys/shopsys/pull/4328
+- updated fixed bar on product detail by @chlebektomas in https://github.com/shopsys/shopsys/pull/4335
+- updated autocomplete popup accessibility and polished design by @chlebektomas in https://github.com/shopsys/shopsys/pull/4230
+- Updated admin UI with responsive layout improvements by @chlebektomas in https://github.com/shopsys/shopsys/pull/4298
+
+### :book: Documentation
+
+- add generate_upgrade_notes Claude slash command by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/4218
+- added docs to correctly describe how to extend input validation in GraphQl types by @TomasLudvik in https://github.com/shopsys/shopsys/pull/4292
+- fix license in README.md by @TomasLudvik in https://github.com/shopsys/shopsys/pull/4301
+- Add copy code functionality to code blocks in documentation by @JanMolcik in https://github.com/shopsys/shopsys/pull/4332
+- added upgrade notes for administration overhaul by @grossmannmartin in https://github.com/shopsys/shopsys/pull/4227
+
+### :rocket: Performance Improvements
+
+- reduced data size transfer for product detail by @chlebektomas in https://github.com/shopsys/shopsys/pull/4324
+- optimized cart sync across tabs by @chlebektomas in https://github.com/shopsys/shopsys/pull/4327
+- order creation: email preparation is now asynchronous by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/4266
+- improve LCP for Homepage by @henzigo in https://github.com/shopsys/shopsys/pull/3953
+
+### :warning: Security
+
+- graphql queries are sent exclusively by POST by @grossmannmartin in https://github.com/shopsys/shopsys/pull/4236
+- replaced jQuery UI with SortableJS by @chlebektomas in https://github.com/shopsys/shopsys/pull/4326
+
+### :up: Dependencies
+
+- disallow installing doctrine-bundle in ">=2.17.0" version by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/4222
+- composer: require doctrine-bundle ^2.18 by @vitek-rostislav in https://github.com/shopsys/shopsys/pull/4226
+- fixed npm versions compatibility by @grossmannmartin in https://github.com/shopsys/shopsys/pull/4223
+- conflicted doctrine/orm 2.20.7 by @grossmannmartin in https://github.com/shopsys/shopsys/pull/4252
+- conflicted webonyx/graphql-php >= 15.29.0 by @grossmannmartin in https://github.com/shopsys/shopsys/pull/4347
```

Comment:
I believe this is not in the released version, but we should have the upgrade of webonyx here, or?